terraform/README.md

```diff
@@ -0,0 +1,31 @@
+# Terraform templates for Event Gateway
```

Comment:
1. Can you move all those file to `/contrib/terraform` folder.
2. Ideally would be to expose it as a module that someone else can use. So I think this readme should be mainly about how to use this module.

Reply:
Not a problem. I'll refactor it in that direction